Puppetfile

```diff
@@ -1,4 +1,4 @@
-forge 'https://forgeapi.puppetlabs.com'
+forge 'https://forgeapi.puppetlabs.com/'
```

Comment:
This is what the script produces. Not sure if this is the best way or if we should update the script.